boards/shields/arduino_modulino_buttons/Kconfig.shield

```diff
@@ -0,0 +1,5 @@
+# Copyright 2025 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config SHIELD_ARDUINO_MODULINO_BUTTONS
+	def_bool $(shields_list_contains,rpi_pico_uno_flexypin)
```

Comment:
Copy-paste mistake?

Reply:
yeah I'll put it on the ever growing list of copy paste messup I did on this one